### ISSUE TITLE
broken apache conf for OBS 2.6 installations on 13.2

### ISSUE THREAD

**Issue by bgeuken** [openSUSE/open-build-service#1156]
I installed the OBS 2.6 rpms on a 13.2 system and found following things:

- passenger didn't got installed, but is required by obs-api. Installing rubygem-passenger-apache2 solved following syntax error when starting apache:
```
Sep 18 16:37:04 freud start_apache2[5128]: AH00526: Syntax error on line 7 of /etc/apache2/vhosts.d/obs.conf:
Sep 18 16:37:04 freud start_apache2[5128]: Invalid command 'PassengerSpawnMethod', perhaps misspelled or defined by a module not included in the server configuration
```

- After fixing above error I ran into:
```
Sep 22 15:04:07 freud start_apache2[22697]: AH00526: Syntax error on line 22 of /etc/apache2/vhosts.d/obs.conf:
Sep 22 15:04:07 freud start_apache2[22697]: Invalid command 'Allow', perhaps misspelled or defined by a module not included in the server configuration
```

That's caused by the "Allow from all" statements that got replaced by "Require all granted" ([apache upgrade guide](http://httpd.apache.org/docs/2.4/upgrading.html)). The current conf file has both, eg.:
```
        <Directory /srv/obs/repos>
           Options Indexes FollowSymLinks
           Allow from all
           Require all granted
        </Directory>
```

Removing the "Allow from all" fixes the error.

**Comment by bgeuken** [comment]
The second issue got fixed now by Adrian: f035033a89cbca2a32

**Comment by bgeuken** [comment]
Other issue got fixed with #1163. Thus closing.
